schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/InterpretationRequestRD.avdl

```diff
@@ -85,10 +75,6 @@ protocol InterpretationRequestsRD {
         */
         union {null, File} pedigreeDiagram;
 
-        /**
-        Variant Annotation using cellbase
-        */
-        union {null, File} annotationFile;
```

Comment:
Don't remove, please keep as we may want to send the annotation. This file is the best one to send actually as it contains the genotypes as well. Do we have a data spec for this file?